README.md

```diff
@@ -220,9 +220,29 @@ Work on ``pandas`` started at AQR (a quantitative hedge fund) in 2008 and
 has been under active development since then.
 
 ## Discussion and Development
-Since pandas development is related to a number of other scientific
-Python projects, questions are welcome on the scipy-user mailing
-list. Specialized discussions or design issues should take place on
-the PyData mailing list / Google group:
+General questions and discussions can take place on the [pydata mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata).
+Specialized discussions or design issues should take place on
+the [pandas-dev mailing list](https://mail.python.org/mailman/listinfo/pandas-dev) / Google group:
 
-https://groups.google.com/forum/#!forum/pydata
+<<<<<<< HEAD
```

Comment:
You seem to have some eft-overs from a rebase / merge conflict (although not sure where this is coming from)

(apart from that the updates look god)